Sprint-3/implement/get-angle-type.js

```diff
@@ -25,3 +25,25 @@
 // Identify Reflex Angles:
 // When the angle is greater than 180 degrees and less than 360 degrees,
 // Then the function should return "Reflex angle"
+
+function getAngleType(angle){
+     if (angle <= 0 || angle >= 360) {
+        return "Invalid angle";
+    }
+    if (angle === 90) {
+        return "Right angle";
+    } else if (angle > 0 && angle < 90) {
```

Comment:
"Angle > 0" is a redundant conditional, as any number less than or equal to 0 has already been dealt with